Slambook/index.js

```diff
@@ -25,8 +26,47 @@ let nickname= document.getElementsByName('nickname');
 let songForMe= document.getElementsByName('songForMe');
 let shareOpinion= document.getElementsByName('shareOpinion');
 
-console.log(contactName,relationship, likeInMe, hateInMe,deadReaction,firstImpression,beautifulMessage,nickname,songForMe,shareOpinion);
-// Write your code above this line
-}
+let inputNames= [contactName, relationship, likeInMe, hateInMe, deadReaction, firstImpression, beautifulMessage, nickname, songForMe, shareOpinion];
+
+let contactNameValue= document.querySelector(`input[name = 'contactName']`).value;
+let relationshipValue= document.querySelector(`[name = 'relationship']`).value;
+let likeInMeValue= document.querySelector(`[name = 'likeInMe']`).value;
+let hateInMeValue= document.querySelector(`[name= 'hateInMe']`).value;

```

Comment:
The 'inputValue' variable is not declared with the 'let' keyword. It should be declared before line 28 to avoid creating an implicit global variable.